install/ansible/playbooks/deploy_cluster.yml

```diff
@@ -26,6 +26,12 @@
       - firewalld
       - libselinux-python
 
+- name: remove any remnant Origin binaries
+  become: true
+  file:
+    state: absent
+    path: "/usr/bin/oc"
+
```

Comment:
[Line 46](https://github.com/RedHat-MultiArch-QE/multiarch-ci-provisioner/pull/22/files#diff-2ddc5c7aaaab29b905104c7a5393d350R46) tells ansible not to overwrite this file in order to prevent unnecessary downloads of the large archive.  I agree that this should definitely be changed to support newer releases of the binary though.  Maybe we could make the download conditional on the oc version matching:

```
- name: check for oc binary
  shell: which oc && echo "true" || echo "false"
  register: oc_exists

- name: check oc version
  shell: oc version | gawk 'NR==1{print $2}'
  when: oc_exists
  register: existing_oc_version

- name: remove oc
  become: true
  file:
    state: absent
    path: /usr/bin/oc
  when: (existing_oc_version is defined) and (existing_oc_version.stdout != oc_version_field)
```

If we did something like that we'd need to also add this to the group vars:
```
oc_version_field: "v{{ openshift_version }}+{{openshift_id }}"
```

Reply:
Updated and tested.